Doc/library/warnings.rst

```diff
@@ -192,7 +192,7 @@ colons::
 
 The meaning of each of these fields is as described in :ref:`warning-filter`.
 When listing multiple filters on a single line (as for
-:envvar:`PYTHONWARNINGS`), the individual filters are separated by commas,and
+:envvar:`PYTHONWARNINGS`), the individual filters are separated by commas and
```

Comment:
The second ``filters take precedence`` that you are quoting is inside the parentheses, so it's not part of the ``and``.  Only the first two verbs that you highlighted are part of the statement.

I did have to read that sentence a few times to understand it, but I didn't want to change any wording in this PR.

Reply:
You are right. I missed parenthesis.

But should not two clauses "filters are separated" and "filters take precedence" be separated by a comma? I am not English expert at all, so sorry for disturbs.

Reply:
I'm glad you're asking.  :-)  In English, sentences with 3 or more verbs should use a comma, but when there are only 2 verbs, there shouldn't be a comma.  Look at the [Comma Between Two Verbs in a Compound Predicate section](https://www.grammarly.com/blog/comma/).

I know there are other instances in the doc where two verbs have a comma before the ``and``.  I fixed this one because there wasn't a space and the correct fix wasn't to add a space, but rather to change the comma into a space.  If you think it's not much code churn, I can fix the other 3 instances on this page.  I don't think it's a big deal to have the comma, but it's technically not correct.